src/Aspire.Dashboard/Configuration/ValidateDashboardOptions.cs

```diff
@@ -100,6 +100,11 @@ public ValidateOptionsResult Validate(string? name, DashboardOptions options)
             }
         }
 
+        if (!options.Frontend.OpenIdConnect.TryParseOptions(out var messages))
+        {
+            errorMessages.AddRange(messages);
+        }
```

Comment:
What about moving this to the `case FrontendAuthMode.OpenIdConnect:` on line 24 above?